en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md

```diff
@@ -230,3 +230,58 @@ You can also navigate to any of the artifacts by using the top-left menu as show
 When you go to the [Analytics portal](#starting-the-analytics-portal) the message details will be logged as follows:
 
 ![Message tracing per API](../assets/img/ei-analytics/119132315/message-tracing.png "Message tracing per API")
+
+## Default Ports
```

Comment:
We need to move this content to https://ei.docs.wso2.com/en/latest/micro-integrator/setup/changing_default_ports/

ALong with some instructions on how to change these ports.

Reply:
Addressed via 535158b13f0bce8d8241c1a81b44a909f5569b2d